### PR TITLE
Blender: Extract Review

### PR DESCRIPTION
**Pull request by simonebarbieri**
## Brief description
Added Review to Blender.

## Description
This implementation is based on #3508 but made compatible for the current implementation of OpenPype for Blender.

### CODE REVIEW

File: openpype/hosts/blender/plugins/publish/integrate_thumbnail.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import pyblish.api
```
Comment (iLLiCiTiT) [inline openpype/hosts/blender/plugins/publish/integrate_thumbnail.py:1]: Import of other plugins is never a good idea. Why is this needed?

**Review by LiborBatek — APPROVED** [review]
Everything working fine...published model and review ok.
image

**Comment by LiborBatek** [comment]
Just a small note, the video doesnt have any burnins like maya one (author, version, date etc.) so not sure how it suppose to work in blender and/or how its created in...

**Comment by m-u-r-p-h-y** [comment]
> Just a small note, the video doesnt have any burnins like maya one (author, version, date etc.) so not sure how it suppose to work in blender and/or how its created in...

It is done by a different plugin - Extract burnins

image

**Review by mkolar — CHANGES_REQUESTED** [review]
There's a syntax error in the default blender settings. That means that you probably created the defaults manually rather than using the developer mode in the settings GUI. Please use it every time you create a new settings to prevent this from happening. 
image

Secondly, it looks like you still have OCIO configs submodule in your branch, however, it was removed in the upstream and this PR is trying to get it back. Please remove it before we merge this.

File: openpype/hosts/blender/plugins/publish/extract_thumbnail.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    Takes review camera and creates a thumbnail based on viewport
+    capture.
+
+    """
+
+    label = "Extract Thumbnail"
+    hosts = ["blender"]
+    families = ["review"]
+    order = pyblish.api.ExtractorOrder + 0.01
+
```
Comment (iLLiCiTiT) [inline openpype/hosts/blender/plugins/publish/extract_thumbnail.py:24]:
```suggestion
    order = pyblish.api.ExtractorOrder + 0.01
    presets = {}
```

File: openpype/hosts/blender/plugins/publish/extract_thumbnail.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        camera = instance.data.get("review_camera", "AUTO")
+        start = instance.data.get("frameStart", bpy.context.scene.frame_start)
+        family = instance.data.get("family")
+        isolate = instance.data("isolate", None)
+
+        project_settings = instance.context.data["project_settings"]["blender"]
+        extractor_settings = project_settings["publish"]["ExtractThumbnail"]
+        presets = extractor_settings.get("presets")
+
+        preset = presets.get(family, {})
```
Comment (iLLiCiTiT) [inline openpype/hosts/blender/plugins/publish/extract_thumbnail.py:43]:
```suggestion
        preset = self.presets.get(family, {})
```

File: openpype/hosts/blender/plugins/publish/extract_playblast.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    hosts = ["blender"]
+    families = ["review"]
+    optional = True
+    order = pyblish.api.ExtractorOrder + 0.01
+
+    def process(self, instance):
+        self.log.info("Extracting capture..")
+
+        self.log.info(instance.data)
+        print(instance.context.data)
```
Comment (iLLiCiTiT) [inline openpype/hosts/blender/plugins/publish/extract_playblast.py:30]:
```suggestion
```

File: openpype/hosts/blender/plugins/publish/extract_thumbnail.py [hunk 5]
```diff
@@ -0,0 +1,99 @@
+import os
+import glob
+
+import pyblish.api
+import openpype.api
```
Comment (iLLiCiTiT) [inline openpype/hosts/blender/plugins/publish/extract_thumbnail.py:5]: Please don't use `openpype.api` anymore.

**Review by antirotor — APPROVED** [review]

**Pull request merged**